### DISCUSSION TITLE
Changes in Web3Auth Developer Dashboard

### DISCUSSION THREAD

**Discussion by yashovardhan** [Web3Auth/Web3Auth#964] [Announcements]
Hey folks,

Recently there have been multiple changes added to the Web3Auth Developer Dashboard. This thread highlights some of the major changes and places it might have affected/ will affect the community at large.

**Client ID Network**
- In some discussions with our community members, we found out that they accidentally were using Web3Auth's `testnet` network in production. This is a huge error since this might result in users losing keys. This is because this network is run on Web3Auth nodes and is meant for testing and beta application usage. 
- To combat this issue, we have added a network section in the generation of Client IDs, and made it mandatory for people to generate a `mainnet` or `cyan` client id in production, according to their choice of network. Please generate a new client id for your preferred production network to enable safe user onboarding experience.
- Related issues: https://github.com/orgs/Web3Auth/discussions/940, https://github.com/orgs/Web3Auth/discussions/897

**Billing Section**
- To enable a seem less, smooth, and transparent experience using Web3Auth, we have introduced the dynamic Billing section, wherein you can track the metrics of how your application has been used over the past cycle and accordingly enable payments using your preferred method.

**Teams**
- Now you can create a team for your organisation and add multiple members to manage your Client IDs and Verifiers seemlessly.

**Interoperability Features**
- One of the most interesting features we've unveiled in the ability to use your Web3Auth generated keys dynamically in Torus Wallet and enable the interoperability for your end user. Have a look at [this documentation](https://web3auth.io/docs/interoperability#using-web3auth-dashboard-toggles) on how you can enable that.

**Comment by dacarva** [comment]
Hi! Is it possible to downgrade `cyan`  to `mainnet` network and preserve the cliendId?
